src/codeu/chat/client/commandline/Chat.java

```diff
@@ -120,11 +126,30 @@ public void invoke(List<String> args) {
         System.out.println("    Add a new user with the given name.");
         System.out.println("  u-sign-in <name>");
         System.out.println("    Sign in as the user with the given name.");
+        System.out.println("  info");
+        System.out.println("    Get session information.");
         System.out.println("  exit");
-        System.out.println("    Exit the program.");
+        System.out.println("    Exit the program.");       
       }
     });
-
+    // info (Server info)
+    //
+    // Get some infomation from server; it should be version info currently
+    //
+    panel.register("info", new Panel.Command() {
```

Comment:
Hmm, it's surprising to me that this is showing up in the changes for this PR. This was already merged into master, so it should not appear as being an addition as part of this PR.

I think this means you need to pull the latest changes from master into this branch. Can you run `git pull origin master`, then `git push`? That should update your branch so we only see the lines that have changed as part of the persistent storage feature.